### PR TITLE
Cawemo 1.9.7

### PR DESCRIPTION
**Pull request by jfriedenstab**
Part of https://github.com/camunda/cawemo/issues/11142

### CODE REVIEW

File: security/content/notices.md [hunk 1]
```diff
@@ -15,6 +15,74 @@ releases of the community platform.
...
+- https://nvd.nist.gov/vuln/detail/CVE-2023-0215
+- https://nvd.nist.gov/vuln/detail/CVE-2023-0216
+- https://nvd.nist.gov/vuln/detail/CVE-2023-0217
+- https://nvd.nist.gov/vuln/detail/CVE-2023-0286
+- https://nvd.nist.gov/vuln/detail/CVE-2023-0401
+- https://nvd.nist.gov/vuln/detail/CVE-2023-0464
+- https://nvd.nist.gov/vuln/detail/CVE-2022-4203
+- https://nvd.nist.gov/vuln/detail/CVE-2022-4304
+- https://nvd.nist.gov/vuln/detail/CVE-2023-0465
+- https://nvd.nist.gov/vuln/detail/CVE-2023-0466
```
Comment (wollefitz) [inline security/content/notices.md:57]: @jfriedenstab We could sort this list to make it easier tracking all the CVE:

```suggestion
- https://nvd.nist.gov/vuln/detail/CVE-2022-4203
- https://nvd.nist.gov/vuln/detail/CVE-2022-4304
- https://nvd.nist.gov/vuln/detail/CVE-2022-4450
- https://nvd.nist.gov/vuln/detail/CVE-2023-0215
- https://nvd.nist.gov/vuln/detail/CVE-2023-0216
- https://nvd.nist.gov/vuln/detail/CVE-2023-0217
- https://nvd.nist.gov/vuln/detail/CVE-2023-0286
- https://nvd.nist.gov/vuln/detail/CVE-2023-0401
- https://nvd.nist.gov/vuln/detail/CVE-2023-0464
- https://nvd.nist.gov/vuln/detail/CVE-2023-0465
- https://nvd.nist.gov/vuln/detail/CVE-2023-0466
```

File: security/content/notices.md [hunk 2]
```diff
@@ -15,6 +15,74 @@ releases of the community platform.
...
+
+- https://nvd.nist.gov/vuln/detail/CVE-2023-23914
+- https://nvd.nist.gov/vuln/detail/CVE-2023-27536
+- https://nvd.nist.gov/vuln/detail/CVE-2023-27533
+- https://nvd.nist.gov/vuln/detail/CVE-2023-27534
+- https://nvd.nist.gov/vuln/detail/CVE-2023-27535
+- https://nvd.nist.gov/vuln/detail/CVE-2023-23915
+- https://nvd.nist.gov/vuln/detail/CVE-2023-23916
+- https://nvd.nist.gov/vuln/detail/CVE-2023-27537
+- https://nvd.nist.gov/vuln/detail/CVE-2023-27538
```
Comment (wollefitz) [inline security/content/notices.md:69]: Same as above

```suggestion
- https://nvd.nist.gov/vuln/detail/CVE-2023-23914
- https://nvd.nist.gov/vuln/detail/CVE-2023-23915
- https://nvd.nist.gov/vuln/detail/CVE-2023-23916
- https://nvd.nist.gov/vuln/detail/CVE-2023-27533
- https://nvd.nist.gov/vuln/detail/CVE-2023-27534
- https://nvd.nist.gov/vuln/detail/CVE-2023-27535
- https://nvd.nist.gov/vuln/detail/CVE-2023-27536
- https://nvd.nist.gov/vuln/detail/CVE-2023-27537
- https://nvd.nist.gov/vuln/detail/CVE-2023-27538
```

File: security/content/notices.md [hunk 3]
```diff
@@ -15,6 +15,74 @@ releases of the community platform.
...
+**Product affected:**
+
+Cawemo On-Premises
+
+**Impact:**
+
+The version of `spring-core` shipped with `cawemo-restapi` was affected by the following vulnerabilities:
+
+- https://nvd.nist.gov/vuln/detail/CVE-2023-20863
+- https://nvd.nist.gov/vuln/detail/CVE-2023-20861
```
Comment (wollefitz) [inline security/content/notices.md:31]: Same as below 

```suggestion
- https://nvd.nist.gov/vuln/detail/CVE-2023-20861
- https://nvd.nist.gov/vuln/detail/CVE-2023-20863
```

**Review by wollefitz — COMMENTED** [review]
lgtm! One small suggestion

File: security/content/notices.md (hunk 1 above)
Comment (jfriedenstab) [reply security/content/notices.md:57]: Thanks, good suggestion. I actually copied the CVEs from the `trivy` output where they were ordered by severity.

**Pull request merged**